src/lib/homestore_backend/hs_homeobject.hpp

```diff
@@ -289,27 +289,31 @@ class HSHomeObject : public HomeObjectImpl {
     struct PGBlobIterator {
         PGBlobIterator(HSHomeObject& home_obj, homestore::group_id_t group_id, uint64_t upto_lsn = 0);
         PG* get_pg_metadata();
-        int64_t get_next_blobs(uint64_t max_num_blobs_in_batch, uint64_t max_batch_size_bytes,
-                               std::vector< HSHomeObject::BlobInfoData >& blob_vec, bool& end_of_shard);
-        void create_pg_shard_snapshot_data(sisl::io_blob_safe& meta_blob);
-        void create_pg_snapshot_data(sisl::io_blob_safe& meta_blob);
-        void create_shard_snapshot_data(sisl::io_blob_safe& meta_blob);
-        void create_blobs_snapshot_data(std::vector< HSHomeObject::BlobInfoData >& blob_vec,
-                                        sisl::io_blob_safe& data_blob, bool end_of_shard);
+        bool updateCursor(objId id);
```

Comment:
Naming format inconsistent with other functions. Let's remember to change it in future PRs.

Reply:
Fixed